externalLibrary/src/Models.elm

```diff
@@ -0,0 +1,104 @@
+module Models exposing (Account, AccountSubType(..), AccountType(..), ExitError, ExitMetadata, Institution, Item, ItemExit, MetadataStatus(..), Model, Product(..), Stage(..), initialModel)
+
+
+type alias Model =
+    { items : List Item
+    , name : String
+    , stage : Stage
+    , itemExit : Maybe ItemExit
+    }
+
+
+initialModel : Model
+initialModel =
+    { items = []
+    , name = ""
+    , stage = Start
+    , itemExit = Nothing
+    }
+
+
+type alias Account =
+    { name : String
+    , id : String
+    , type_ : AccountType
+    , subType : AccountSubType
+    , mask : String
+    , primary : Bool
+    }
+
+
+type AccountType
+    = Depository
+    | Credit
+    | Brokerage
+    | Loan
+    | OtherType
+
+
+type AccountSubType
+    = CD
+    | Checking
+    | Savings
+    | MoneyMarket
+    | Paypal
+    | Prepaid
+    | CreditCard
+    | Rewards
+    | OtherSubType
+
+
+type alias Institution =
+    { name : String
+    , id : String
+    }
+
+
+type alias Item =
```

Comment:
As someone who isn't familiar with Plaid, I struggled a lot with `Item` as a name. It doesn't carry enough meaning to tell me much about what kind of data is in it until I get here to read it. Would something like `BankDetails` make sense as an alternate name?